packages/fxa-profile-server/lib/routes/profile.js

```diff
@@ -71,13 +71,13 @@ module.exports = {
       }
       const lastModified = cached ? new Date(cached.stored) : new Date();
       if (cached) {
-        logger.info('batch.cached', {
+        logger.trace('batch.cached', {
```

Comment:
this logging is important to make sure that profile server caching is working. If we make this `trace`, we just want to make sure that we will be able to tell if something breaks in production w.r.t to the cache. It can also be that we haven't monitored this logging since Kibana days.

Reply:
I don't think we have.  jrgm mentioned in the issue that statsd would be a better alternative to logger if getting exact cache numbers was important.

Does watching general cache metrics answer the questions you'd have if we made this change?